### ISSUE TITLE
Analytics connector doesn't pass in profile id

### ISSUE THREAD

**Issue by decthomas** [forkcms/forkcms#1944]
## Type

- Critical bug

## Problem description

After having coupled an Analytics account in the backend (Settings > Modules > Analytics), the backend dashboard shows the following error: 
`Error calling GET https://www.googleapis.com/analytics/v3/data/ga?ids=ga%3A&start-date=2017-01-11&end-date=2017-01-18&metrics=ga%3Apageviews%2Cga%3Ausers%2Cga%3ApageviewsPerSession%2Cga%3AavgSessionDuration%2Cga%3ApercentNewSessions%2Cga%3AbounceRate: (400) Invalid value 'ga:'. Values must match the following regular expression: 'ga:[0-9]+'`
[Full stack trace here](https://gist.github.com/decthomas/109e50c3700fb7d84239026231f27669)

Passing in the profile id manually [here](https://github.com/forkcms/forkcms/blob/4.4.1/src/Backend/Modules/Analytics/GoogleClient/Connector.php#L349) fixes the issue.

## Steps to reproduce

Hook up your Analytics account in the backend via the Settings > Modules > Analytics tab.
Now navigate to the dashboard, you should see the error.

## Expected behavior / Proposed solutions

The dashboard should load, showing the available widgets, including the analytics one.

**Comment by carakas** [comment]
@decthomas would you be able to provide us with a PR for this?

**Comment by decthomas** [comment]
Looked into it and turns out I had to keep clicking "Save" until the button disappeared because the coupling wasn't entirely finished.

I stopped clicking "Save" when it said "Coupled profile: UA-XXXXXXX-XX" at the top which made it seem as if the coupling had succeeded. It might be nice to actually ask the user to perform an action so that it's clear what is expected.

I'll close this issue; feel free to add an improvement issue if you agree with what I said above.
